sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/SampleTest.java

```diff
@@ -1,42 +0,0 @@
-package com.azure.digitaltwins.core;
```

Comment:
I'm deleting this sample test because we don't need it anymore. The model tests will serve as the sample tests for now